Documentation/container-linux.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Container Linux
+weight: 80
+indent: true
+---
+# Using the Container Linux Update Operator with Rook
+
+When you are using Container Linux (CoreOS) and have the update engine enabled, it could be that a node reboots quickly after another not leaving enough time for the Rook cluster to rebuild. The [Container Linux Update Operator](https://github.com/coreos/container-linux-update-operator) is the solution for this, you can block your nodes to reboot until the Ceph cluster is healthy.
+
+## Prequisites
+- An operational Container Linux Kubernetes cluster (Successfully tested with 1.8.4)
+- A working rook cluster
+- The update-engine.service systemd unit on each machine should be unmasked, enabled and started in systemd
+- The locksmithd.service systemd unit on each machine should be masked and stopped in systemd
+
+## Start the update operator
+
+Proper reading of the README on the [Container Linux Update Operator](https://github.com/coreos/container-linux-update-operator) is necessary. Clone the repo and go in the `examples` directory
+
+Look for the file named `update-operator.yaml` and add following lines to the `command` part of the container
+
+```
+- "--before-reboot-annotations"
+- "ceph-reboot-check"
+- "-v"
```

Comment:
Could you remove the `-v6` flag (and sentence about it below) as this is about the `--*-reboot-annotations` flag?